src/envs/pybullet_robots.py

```diff
@@ -294,15 +320,124 @@ def run_inverse_kinematics(self, end_effector_pose: Pose3D,
                                   physics_client_id=self._physics_client_id,
                                   validate=validate)
 
+    def create_move_end_effector_to_pose_option(
+        self,
+        name: str,
+        types: Sequence[Type],
+        params_space: Box,
+        get_current_and_target_pose_and_finger_status: Callable[
+            [State, Sequence[Object], Array], Tuple[Pose3D, Pose3D, str]],
+    ) -> ParameterizedOption:
+        """Fingers drift if left alone.
```

Comment:
this docstring doesn't make sense here, and we shouldn't have one anyway since the parent method has one. move this to where we say "Handle fingers."?